biztalk/core/high-availability-using-sql-server-always-on-availability-groups.md

```diff
@@ -139,14 +128,15 @@ Replace `‘dbname’` with the corresponding database name against which the jo
 1. Check your OS requirements: 
 2. On all **Windows Server 2012 R2** computers, install the [3090973 MSDTC hotfix](https://support.microsoft.com/kb/3090973) (opens a KB article)
 3. On all **Windows Server 2016** computers, enable the [RemoteAccessEnabled registry key](https://support.microsoft.com/kb/3182294) (opens a KB article)
-4. Make sure to have at least four different SQL instances which will host the various BizTalk databases. The secondary replicas should also be set up on different SQL instances. This results in a minimum of 8 SQL instances (1 primary and 1 secondary replica for each of the 4 instances), and a minimum of 4 Availability Groups. See the above illustration for this Availability Group configuration. Make sure Availability Groups are created with the **Per Database DTC Support** option as this cannot be changed later. 
+4. Create required Availability Groups. Make sure Availability Groups are created with the **Per Database DTC Support** option. 
 5. When configuring BizTalk Server and specifying the SQL server name, use the Availability Group’s listener name instead of the actual machine name. This creates the BizTalk databases, logins, and SQL Agent jobs on the current primary replica. 
 6. Stop BizTalk processing (Host Instances, SSO Service, IIS, Rules Engine Update Service, BAMAlerts Service, and so on), and stop the SQL Agent Jobs. 
-7. Now add BIzTalk databases to the respective Availability Groups. 
+7. Now add BizTalk databases to the respective Availability Groups. 
 8. Enclose body of SQL Agent job steps within `IF` block (mentioned previously) to make sure they run only if the target is the primary replica. 
 9. Script Logins and SQL Agent Jobs to replicate them on corresponding replica. 
 10. Replicate SQL DBMail Profile and Account for BAM Alerts on corresponding SQL instances hosting the secondary replica. 
 11. If you are adding an additional message box database or deploying a new BAM activity/view later, then new SQL jobs are created for new message box databases or BAM Alerts database on the current primary replica. Make sure to edit it on primary replica, and then create them manually on the corresponding secondary replicas. 
+12. Starting with BizTalk Server 2020, BAM DTS packages are deployed to SSIS Catalog. Add SSISDB database to availability group alongwith BizTalk databases. Use steps provided in [AlwaysON for SSIS Catalog](https://docs.microsoft.com/en-us/sql/integration-services/catalog/ssis-catalog?view=sql-server-ver15#always-on-for-ssis-catalog-ssisdb).
```

Comment:
Noticed in some other docs that "Starting with BizTalk Server 2020" is written in bold, and sometimes also as a note.
Not sure if it makes sense to the same here as well